src/check_cve/check.py

```diff
@@ -46,8 +51,8 @@ def is_vulnerable(host):
     logging.debug("Disabling URL normalization.")
     urllib3.util.url.NORMALIZABLE_SCHEMES = ()
 
-    logging.debug("Creating connection pool")
-    pool = urllib3.PoolManager(cert_reqs="CERT_NONE")
+    logging.debug(f"Creating connection pool, retries={retries}, timeout={timeout}")
```

Comment:
This is minor, but as long as we're logging, might as well be nice and mention that the timeout is in seconds

Reply:
Good idea.  I was going to add `seconds` to the message but it was going to force the code to wrap.  Came up with a good solution in: 9d42ffc

Reply:
I like it. 👍